br-ext/package/optee_rust_examples_ext/Config.in

```diff
@@ -11,6 +11,12 @@ config BR2_PACKAGE_OPTEE_RUST_EXAMPLES_EXT_SITE
 
 if BR2_PACKAGE_OPTEE_RUST_EXAMPLES_EXT
 
+config BR2_PACKAGE_OPTEE_RUST_EXAMPLES_EXT_SDK
+	string "OPTEE SDK path"
+	default ""
+	help
+	  some help
```

Comment:
Please add some help. "The path to the SDK generated by the optee_rust project" for example?

Reply:
Actually it rather points to `OPTEE_OS_TA_DEV_KIT_DIR` needed to build rust examples. It is defined on similar lines as `BR2_PACKAGE_OPTEE_EXAMPLES_EXT_SDK`. So how about instead "The path to OP-TEE TA development SDK"?

Reply:
Yes, sounds good!